server/src/main/java/org/tbbtalent/server/service/db/impl/CandidateServiceImpl.java

```diff
@@ -61,120 +34,50 @@
 import org.springframework.transaction.annotation.Transactional;
 import org.springframework.web.reactive.function.client.WebClientException;
 import org.tbbtalent.server.configuration.GoogleDriveConfig;
-import org.tbbtalent.server.exception.CountryRestrictionException;
-import org.tbbtalent.server.exception.EntityExistsException;
-import org.tbbtalent.server.exception.EntityReferencedException;
-import org.tbbtalent.server.exception.ExportFailedException;
-import org.tbbtalent.server.exception.InvalidRequestException;
-import org.tbbtalent.server.exception.InvalidSessionException;
-import org.tbbtalent.server.exception.NoSuchObjectException;
-import org.tbbtalent.server.exception.PasswordMatchException;
-import org.tbbtalent.server.exception.SalesforceException;
-import org.tbbtalent.server.exception.UsernameTakenException;
-import org.tbbtalent.server.model.db.Candidate;
-import org.tbbtalent.server.model.db.CandidateDestination;
-import org.tbbtalent.server.model.db.CandidateEducation;
-import org.tbbtalent.server.model.db.CandidateExam;
-import org.tbbtalent.server.model.db.CandidateLanguage;
-import org.tbbtalent.server.model.db.CandidateOccupation;
-import org.tbbtalent.server.model.db.CandidateProperty;
-import org.tbbtalent.server.model.db.CandidateStatus;
-import org.tbbtalent.server.model.db.CandidateSubfolderType;
-import org.tbbtalent.server.model.db.Country;
-import org.tbbtalent.server.model.db.DataRow;
-import org.tbbtalent.server.model.db.DependantRelations;
-import org.tbbtalent.server.model.db.EducationLevel;
-import org.tbbtalent.server.model.db.Exam;
-import org.tbbtalent.server.model.db.Gender;
-import org.tbbtalent.server.model.db.HasTcQueryParameters;
-import org.tbbtalent.server.model.db.LanguageLevel;
-import org.tbbtalent.server.model.db.Occupation;
-import org.tbbtalent.server.model.db.PartnerImpl;
-import org.tbbtalent.server.model.db.QuestionTaskAssignmentImpl;
-import org.tbbtalent.server.model.db.Role;
-import org.tbbtalent.server.model.db.RootRequest;
-import org.tbbtalent.server.model.db.SavedList;
-import org.tbbtalent.server.model.db.SavedSearch;
-import org.tbbtalent.server.model.db.SearchJoin;
-import org.tbbtalent.server.model.db.Status;
-import org.tbbtalent.server.model.db.SurveyType;
-import org.tbbtalent.server.model.db.TaskAssignmentImpl;
-import org.tbbtalent.server.model.db.UnhcrStatus;
-import org.tbbtalent.server.model.db.UploadTaskImpl;
-import org.tbbtalent.server.model.db.User;
-import org.tbbtalent.server.model.db.YesNoUnsure;
+import org.tbbtalent.server.exception.*;
```

Comment:
I lean towards not using wildcard imports i.e. switching that off in intellij

Reply:
Me too. This was done on another machine where I haven't configured Intellij the way we want it.

This is a good example of why it would be good get our standard Intellij configuration into the repo. As I say, I have tried to get that working, but failed so far.

Reply:
Ok thanks John - can pick up on the Intellij shared config separately - good idea.